cmd/nitro/init.go

```diff
@@ -475,6 +475,51 @@ func validateOrUpgradeWasmStoreSchemaVersion(db ethdb.Database) error {
 	return nil
 }
 
+func rebuildLocalWasm(ctx context.Context, config *NodeConfig, l2BlockChain *core.BlockChain, chainConfig *params.ChainConfig, chainDb, wasmDb ethdb.Database, rebuildMode string) (ethdb.Database, *core.BlockChain, error) {
```

Comment:
chainConfig can be taken from l2BlockChain
Instead of NodeConfig, pass only the gethexec.Config which will make it easier to move it there